deployments/k8s-v1.16/sriovdp-daemonset.yaml

```diff
@@ -33,7 +37,7 @@ spec:
       containers:
       - name: kube-sriovdp
         image: nfvpe/sriov-device-plugin
-        imagePullPolicy: Never
+        imagePullPolicy: IfNotPresent
```

Comment:
nit: the `imagePullPolicy` is different from `v1.10-v1.15` version, shall we use the same `Never`?